cmd/podman/create.go

```diff
@@ -90,6 +90,10 @@ func createCmd(c *cli.Context) error {
 	storageOpts.UIDMap = mappings.UIDMap
 	storageOpts.GIDMap = mappings.GIDMap
 
+	if os.Getuid() != 0 {
```

Comment:
euid?

Reply:
yes thanks, that better.  `os.Getuid()` is akready used in few different places, I pushed a new patch to fix it everywhere